letsdrill.rb

```diff
@@ -1,24 +1,31 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  if integer >= 90
+    return "A"
+  elsif integer >= 80
+    return "B"
+  elsif integer >= 70
+    return "C"
+  elsif intger >= 60
+    return "D"
+  else
+    return "F"
+  end
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+  array.sort_by! { |n| n.length }
```

Comment:
Would you believe Enumerable also provides a `min_by` method? Scan down through this page, there's a bunch of good stuff in there: http://ruby-doc.org/core-2.2.2/Enumerable.html